### PR TITLE
guardian signature

### PR DESCRIPTION
**Pull request by AdrianAndersen**
- **feat: guardian signature emails**
- **build: add .nvmrc**
- **1.7.3**
- **feat: add email template for guardian signature**
- **1.7.4**

### CODE REVIEW

**Comment by AdrianAndersen** [comment]
[Legge til rutine for utsending av epost + SMS med lenke til signering for foresatte](https://trello.com/c/hLCSEFZE/903-legge-til-rutine-for-utsending-av-epost-sms-med-lenke-til-signering-for-foresatte)

**Comment by AdrianAndersen** [comment]
See https://github.com/boklisten/bl-api/pull/496

File: src/pages/guardian-signature.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+<spacer size="8"></spacer>
+
+<container>
+  <row>
+    <columns small="12" class="bl-container">
+      <p class="bl-center-text">
+        Du mottar denne eposten fordi du er satt opp som foresatt for {{i
+        'emailTemplateInput.userFullName'}}, som har bestilt bøker hos
+        Boklisten.no. Ettersom {{i 'emailTemplateInput.userFullName'}} er under
```
Comment (LarsSelbekk) [inline src/pages/guardian-signature.html:21]: Branch name

**Review by LarsSelbekk — CHANGES_REQUESTED** [review]

File: src/pages/guardian-signature.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+<spacer size="8"></spacer>
+
+<container>
+  <row>
+    <columns small="12" class="bl-container">
+      <p class="bl-center-text">
+        Du mottar denne eposten fordi du er satt opp som foresatt for {{i
+        'emailTemplateInput.userFullName'}}, som har bestilt bøker hos
+        Boklisten.no {{i 'emailTemplateInput.extra.branchName'}}. Ettersom {{i
```
Comment (LarsSelbekk) [inline src/pages/guardian-signature.html:21]: I would prefer a little more fancy formatting, e.g. "bestilt bøker fra Ullern VGS gjennom Boklisten.no", and if branch not set, "bestilt bøker fra Boklisten.no". The way you currently have it, there will be an extra space in front of the period if the branch name is not set.

**Review by LarsSelbekk — CHANGES_REQUESTED** [review]

**Review by LarsSelbekk — APPROVED** [review]

**Pull request merged**